### ISSUE TITLE
2,800+ employees shown with no supervisor in the app

### ISSUE THREAD

**Issue by Travis-A-Clark** [bcgov/performance#717]
We don't seem to be processing all of the correct reporting relationships from PeopleSoft. There are hundreds of blanks in the "reports to" column of the Statistics and Reports generated in the PDP. When I look at those user profiles, their homepage says their current supervisor is "no supervisor". This isn't the case in other systems that also rely on PeopleSoft data. Here's a list of all 2,877: [https://app.zenhub.com/files/329107952/ccb165d0-8244-436b-ad98-d7f2c1bd130a/download](https://app.zenhub.com/files/329107952/ccb165d0-8244-436b-ad98-d7f2c1bd130a/download)

See below for an example.

In MyPerformance, Barbara Quelch reports to Kimberly Steinmetz:
![image.png](https://images.zenhubusercontent.com/5ffe1f472e351dd9b643025b/6f23dee2-281b-4ea2-8d28-16c7c1211e75)
<br>

In PDP, Barbara Quelch report to No Supervisor:

![image.png](https://images.zenhubusercontent.com/5ffe1f472e351dd9b643025b/aea3b29d-7d0c-446e-85e1-b0b88fd632b2)

**Comment by telusdcinco** [comment]
Checking Prod employee_demo, there is no person currently assigned to the employee's supervisor_position_number.  This could be a result of positional changes for the employee or the supervisor.

We can forward a sample list of employees with blank supervisor details to BI Team or HR to confirm current employee status.

What would be the expectation in case when the supervisor position is vacated, and no other person is assigned to it?  Do we keep the previous supervisor details so that the employee remains to have the last supervisor even though it may no longer be correct?

The current ODS Data Pull process checks on employee ID and record number to match an existing row in employee_demo table, and updates all the other fields accordingly.

**Comment by Travis-A-Clark** [comment]
@telusdcinco Yes, let's send the list of 2,800 that I attached to this ticket along to BI for confirmation. It seems like a very high number.

In terms of expectations when reporting to a vacant position, MyPerformance currently defaults to the employee reporting to the next person above the vacant box in the hierarchy. I created ticket #714 to deal with this as well. It still seems awfully high for 2,800 people to be reporting to vacant positions.

Let me know what you find! Thanks.
